reader/feed/handler.go

```diff
@@ -131,6 +130,22 @@ func (h *Handler) RefreshFeed(userID, feedID int64) error {
 		logger.Debug("[Handler:RefreshFeed] Feed #%d not modified", feedID)
 	}
 
+	if originalFeed.TitleFilter != "" {
+		if storeErr := h.store.FilterByTitle(originalFeed.UserID, originalFeed.ID, originalFeed.TitleFilter); storeErr != nil {
```

Comment:
This is going to generate a lot of SQL queries. What about doing the filtering only when the feed is modified just above?

Perhaps, it could be part of the function `UpdateEntries()`...

Reply:
Good idea~

I have move it into the `UpdateEntries` function